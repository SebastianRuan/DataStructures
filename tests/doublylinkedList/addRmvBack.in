a 5 0
a 12 0
a 333 0
a 99 0
f 17
a 17 3
f 17
a 11 5
a 8 6
r 6
a 200 5
r 5
f 11
