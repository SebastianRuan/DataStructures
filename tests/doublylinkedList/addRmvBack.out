f d:17 i:-1
f d:17 i:3
f d:11 i:5
99
333
12
17
5
11
